src/main/java/jarvis/model/MasteryCheck.java

```diff
@@ -44,4 +44,9 @@ public int hashCode() {
     public String toString() {
         return "Mastery Check at " + getTimePeriod();
     }
+
+    @Override
+    public String getLessonType() {
+        return "Mastery Check";
```

Comment:
Could use `.getClass.getSimpleName()`  to replace this method unless we expect to format the lesson name more next time.

Reply:
I mentioned something similar in #92, alternatively we can also use an enum in case we need to change the string in the future